### PR TITLE
 im out of breadth

### PR DESCRIPTION
**Pull request by S2Mackinley**

### CODE REVIEW

File: javascript/code-challenges/tree/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- [x] Top-level README “Table of Contents” is updated
+- [x] Feature tasks for this challenge are completed
+- [x] Unit tests written and passing
+  - [x] “Happy Path” - Expected outcome
+  - [x] Expected failure
+  - [x] Edge Case (if applicable/obvious)
+- [x] README for this challenge is complete
+  - [x] Summary, Description, Approach & Efficiency, Solution
+  - [x] Link to code
+  - [] Picture of whiteboard
```
Comment (lizkavalski) [inline javascript/code-challenges/tree/README.md:17]: this go in the pull request.
